modules/issue_tracker/test/Issue_TrackerTest.php

```diff
@@ -103,7 +100,7 @@ function testIssueTrackerDoespageLoad()
      */
     function testIssueTrackerDoespageLoadWithPermission()
     {
-        $this->setupPermissions(array("issue_tracker_reporter"));
+        $this->setupPermissions(array("access_all_profiles"));
```

Comment:
Looks like a regression in the required permission?

Reply:
Also unsure. @kongtiaowang do you have any insight?

Reply:
Apparently the permission had to be changed so that the PR could pass travis.